src/Shared/EventArgsFormatting.cs

```diff
@@ -168,7 +213,8 @@ internal static string FormatEventMessage
             int endLineNumber,
             int columnNumber,
             int endColumnNumber,
-            int threadId
+            int threadId,
+            string LogOutputProperties
```

Comment:
Style nit: local variables and method arguments get lower-case names

```suggestion
            string logOutputProperties
```